modules/programs/git.nix

```diff
@@ -83,6 +105,18 @@ in
         example = [ "*~" "*.swp" ];
         description = "List of paths that should be globally ignored.";
       };
+
+      includes = mkOption {
+        type = types.listOf includeModule;
+        default = [];
+        example = literalExample ''
+          [
+            { path = "~/path/to/config.inc"; }
+            { path = "~/path/to/conditional.inc"; condition = "gitdir:~/src/dir"; }
```

Comment:
I'd propose to change the example to something like
```nix
[
  { path = "~/path/to/config.inc"; }
  {
    path = "~/path/to/conditional.inc";
    condition = "gitdir:~/src/dir";
  }
]
```
so that it doesn't get too long in the man page.